src/tools/execution-results.h

```diff
@@ -75,7 +78,17 @@ struct LoggingExternalInterface : public ShellExternalInterface {
       } else if (import->base == "getTempRet0") {
         return {Literal(state.tempRet0)};
       }
+    } else if (import->module == Intrinsics::BinaryenIntrinsics) {
```

Comment:
Should this be handled in the core interpreter instead of in the external interface? Since it's intrinsic, I would expect the core interpreter to know about it and be able to handle it.

Reply:
Yeah, as an intrinsic that seems to make sense. OTOH the mechanism is an import call, and those are not handled in the core interpreter. For now I think avoiding import handling in the core interpreter is best. Later if we refactor intrinsics to something nicer without an import then the core interpreter would be the right place, I agree.

Reply:
Agree we shouldn't try to handle real imports in the core interpreter, but I don't think we should consider this a real import. Given we need to special case intrinsics either way, is there any benefit to not handling them in the core interpreter? I don't know if we have any optimizations that interpret across call boundaries, but in principle handling this in core would make those optimizations more robust.

Reply:
You're right that they could benefit in theory, but no existing pass (or planned one AFAIK) does that.

The benefit to not adding import handling in the core interpreter is just to keep it a little simpler for now.

Reply:
I still think it's simpler to handle this in the core interpreter, but if you feel strongly about it that's ok with me.

Reply:
I don't feel strongly. Changed to how you prefer.